.gitignore

```diff
@@ -48,4 +48,4 @@ supervisord.pid
 tmp/*
 
 # private projects
-addons-yara
+private/
```

Comment:
It's fine, but it will break yara locally while https://github.com/mozilla/addons-server/pull/12059 is not merged, so maybe only add `private/` in this PR, without removing `addons-yara` ? No big deal though, we can handle that in #12059 shortly.

Reply:
ah yes, good catch, thanks